velox/expression/EvalCtx.cpp

```diff
@@ -127,6 +127,7 @@ void EvalCtx::ensureErrorsVectorSize(ErrorVectorPtr& vector, vector_size_t size)
   if (!vector) {
     vector = std::make_shared<ErrorVector>(
         pool(),
+        OpaqueType::create<void>(),
```

Comment:
I guess we should also remove this at some point since Opaque shouldn't also be a logical type?

Reply:
Yup.